types/office-js-preview/index.d.ts

```diff
@@ -18491,14 +18547,266 @@ declare namespace Office {
+         *                 which is an `Office.AsyncResult` object. The sensitivity labels catalog status is returned in the `asyncResult.value` property.
+         * 
+         * @beta
+         */
+        getIsEnabledAsync(options: Office.AsyncContextOptions, callback: (asyncResult: Office.AsyncResult<boolean>) => void): void;
+        /**
+         * Checks whether the sensitivity labels catalog is enabled in Outlook.
+         * 
+         * @remarks
+         * [Api set: Mailbox preview]
+         * 
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read/write item**
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
+         * 
+         * **Important**: The sensitivity labels catalog is configured by an organization's administrator. For more information, see
+         * {@link https://learn.microsoft.com/microsoft-365/compliance/get-started-with-sensitivity-labels | Get started with sensitivity labels}.
+         * 
+         * @param callback - When the method completes, the function passed in the `callback` parameter is called with a single parameter, `asyncResult`,
+         *                 which is an `Office.AsyncResult` object. The sensitivity labels catalog status is returned in the `asyncResult.value` property.
+         * 
+         * @beta
+         */
+        getIsEnabledAsync(callback: (asyncResult: Office.AsyncResult<boolean>) => void): void;
+    }
+    /**
+     * Provides the change status of the sensitivity label applied to a message or appointment in compose mode. This information is provided when the
+     * `Office.EventType.SensitivityLabelChanged` event is raised.
+     * 
+     * @remarks
+     * [Api set: Mailbox preview]
+     * 
+     * @beta
+     */
+    export interface SensitivityLabelChangedEventArgs {
+        /**
+         * The type of event that was raised. For details, refer to {@link https://learn.microsoft.com/javascript/api/office/office.eventtype | Office.EventType}.
+         * 
+         * @remarks
+         * [Api set: Mailbox preview]
+         * 
+         * @beta
+         */
+        type: "olkSensitivityLabelChanged";
+    }
+    /**
+     * Represents the properties of available sensitivity labels in Outlook.
+     * 
+     * @remarks
+     * [Api set: Mailbox preview]
+     * 
+     * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read/write item**
+     *
+     * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
+     * 
+     * @beta
+     */
+    interface SensitivityLabelDetails {
+        /**
+         * The sensitivity label's color.
```

Comment:
I think we try to avoid possessives where it'll still sound okay. If you do take this suggestion, please change the other properties in this interface.
```suggestion
         * The color of the sensitivity label.
```